ocs_ci/utility/ibm_hci.py

```
@@ -62,6 +78,276 @@ def _load_rack_details(self):
+        The primary file is then updated so future runs don't need this step.
+        """
+        from ocs_ci.framework import config
+
+        cluster_name = config.ENV_DATA.get("cluster_name", "")
+
+        # Collect all candidate backup files from both locations, newest first
+        candidates = []
+
+        rack_dir = Path(constants.IBM_HCI_RACK_DIR)
+        for p in rack_dir.glob(f"{cluster_name}_backup_*.json"):
+            candidates.append(p)
```

Ruff (0.15.20)

[warning] Use `list` or `list.copy` to create a copy of a list

(PERF402)